crates/node/src/lib.rs

```diff
@@ -291,12 +300,14 @@ impl Node {
                 PARTITION_TABLE_KEY.clone(),
                 |partition_table| {
                     if let Some(partition_table) = partition_table {
+                        info!("Retrieved partition table: {partition_table:?}");
                         Operation::Return(partition_table)
                     } else {
-                        Operation::Upsert(FixedPartitionTable::new(
-                            Version::MIN,
-                            options.worker.partitions,
-                        ))
+                        let partition_table =
+                            FixedPartitionTable::new(Version::MIN, options.worker.partitions);
+
+                        info!("Initializing a new partition table: {partition_table:?}",);
```

Comment:
Not sure about the value of printing the partition value's debug string.

Reply:
It is actually annoying on info level. I changed it in some follow-up PR to `debug!`. Will change it here.